tests/resolvers/Mutation/adminRemoveEvent.spec.ts

```diff
@@ -167,7 +167,7 @@ describe("resolvers -> Mutation -> adminRemoveEvent", () => {
       context
     );
 
-    expect(adminRemoveEventPayload).toEqual(testEvent!.toObject());
+    expect(adminRemoveEventPayload?._id).toEqual(testEvent!._id);
```

Comment:
The purpose here is to check that the fields you wanted to mutate were mutated. Checking just the `_id` won't give you that.